components/Steps/Confirmation.jsx

```diff
@@ -1,45 +1,33 @@
 import PropTypes from 'prop-types';
 import { useRouter } from 'next/router';
+import Button from 'components/Button/Button';
 
-import Summary from 'components/Summary/Summary';
-
-const ConfirmationStep = ({
-  formData,
-  formSteps,
-  formPath,
-  successMessage,
-  isSummaryCollapsable,
-}) => {
+const ConfirmationStep = ({ formSteps, successMessage }) => {
   const router = useRouter();
-  const { ref } = router.query;
+  const { id } = router.query;
   if (!formSteps) return null;
   return (
-    <div>
+    <>
+      <h1 className="govuk-fieldset__legend--l gov-weight-lighter govuk-!-margin-bottom-7">
+        {successMessage || 'Submission complete'}
+      </h1>
       <div className="govuk-panel govuk-panel--confirmation govuk-!-margin-bottom-9">
-        <h1 className="govuk-panel__title">
-          {successMessage || 'Submission complete'}
+        <h1 className="govuk-fieldset__legend--l gov-weight-lighter">
+          {formSteps[0].title} details have been added
         </h1>
-        {ref && (
-          <div className="govuk-panel__body">
-            Your reference code
-            <br />
-            <strong>{ref}</strong>
-          </div>
-        )}
       </div>
-      <Summary
-        formData={formData}
-        formPath={formPath}
-        formSteps={formSteps}
-        isSummaryCollapsable={isSummaryCollapsable}
-      />
-    </div>
+      <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+        <Button label="Return to search" isSecondary wideButton route={`/`} />
+        <Button label="View person" wideButton route={`/people/${id}`} />
```

Comment:
This is wrong and we fixed it already on https://github.com/LBHackney-IT/lbh-social-care/pull/213. This is assuming that all the forms are related to residents.

Reply:
<img width="496" alt="Screenshot 2021-03-09 at 12 15 52" src="https://user-images.githubusercontent.com/9123541/111210676-70a75c80-85c5-11eb-84ee-a49903365ac2.png">
Joy's new confirmation designs. Should we create a new confirmation page?